flexmemb.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+// chapter 14 exercise 12
+
+typedef struct flex
+{
+    size_t numberOfScores;
+    double averageScore;
+    double scores[];
+}Flex_t;
+
+static void printStructValue(const struct flex * flexStruct)
+{
+    printf("Scores : ");
+    for(int index = 0; index < flexStruct->numberOfScores; index++)
+    {
+        printf("%g ", flexStruct->scores[index]);
+    }
+    printf("\nAverage: %g\n", flexStruct->averageScore);
+}
+
+static struct flex * assignFlexStructValues(int numberOfScores, double baseValue)
+{
+    double sumOfScores = 0;
+    struct flex * flexStruct1;
```

Comment:
minor naming flexStruct1 --> implies there is a flexStruct 2?  why not just flexStruct